### ISSUE TITLE
PhysioQA

### ISSUE THREAD

**Issue by RickReddy** [brainhackorg/global2023#87]
### Title

PhysioQA

### Leaders

@RickReddy - Rithwik Guntaka

### Collaborators

@rgbayrak - Roza Bayrak

### Brainhack Global 2023 Event

BrainHack Vanderbilt

### Project Description

We are working on creating a model that can classify physiological data (respiratory + cardiac) that is associated with fMRI data, so that the end user can determine whether the data is usable, if it needs to be modified to be usable, or if it is simply not usable.

When it comes to using peripheral physiological data in your fMRI data analysis, the quality of the recordings is super important, but let's face it, checking the quality of this data can be a real headache. It usually involves a lot of manual work and you need to know what is real data, what is an artifact. That's why we want to create a nifty deep-learning tool to automate quality assessment! This tool doesn't just check the quality of your data; it also points out any issues and gives you tips on how to fix them. It's like having a friendly expert on your team, making sure your research data is as good as it can be!

### Link to project repository/sources

https://github.com/brainhack-vandy/projects/physioQA

### Goals for Brainhack Global

- Brainstorm different techniques to increase the classification accuracy of the model
- Modify the MATLAB GUI to be more usable for classifying data
- Potentially, getting data labeled by an expert who is familiar with physiological data

### Good first issues

Classification tool (beginner machine learning friendly)
- experimenting with different neural network architectures using keras
- experimenting with feature engineering
- experiment with different hyperparameters for the model
Manual annotation tool
- adding more button functionality to the GUI tool, to allow for more detailed labeling of data
- modify the GUI to select and label certain sections of data

### Communication channels

#physioqa channel on https://discord.gg/GyeeVbYC

### Skills

Having any one of these skills would enable an individual to contribute. However, if they have none of these there are onboarding documents that would help them experiment, learn, and contribute regardless.
- Familiarity with Python and Jupyter notebooks
- Familiarity with MATLAB
- Familiarity with physiological data in order to asses its quality

### What will participants learn?

Participants will:
- Learn the significance and influence of physiological data in fMRI analysis
- Become familiar with and explore different aspects of ML, and how it can be used for timeseries data
- Learning how to create/modify a GUI to analyze timeseries data using MATLAB toolbox

### Data to use

Public HCP dataset that has physiological data paired with fMRI data.

https://www.humanconnectome.org/study/hcp-young-adult

### Number of collaborators

3

### Credit to collaborators

Collaborators will be credited on the GitHub site and credited in any paper that results from this project

### Image

<img width="842" alt="MicrosoftTeams-image (1)" src="https://github.com/brainhackorg/global2023/assets/40832092/29c99dbf-8d4c-4515-bed4-a4cf4b6c5d72">

### Type

method_development, pipeline_development, visualization

### Development status

1_basic structure

### Topic

data_visualisation, deep_learning, machine_learning, physiology

### Tools

Jupyter

### Programming language

Matlab, Python

### Modalities

fMRI, other

### Git skills

0_no_git_skills, 1_commit_push, 2_branches_PRs

### Anything else?

other under modalities: physiological data (cardiac + respiration)

### Things to do after the project is submitted and ready to review.

- [ ] Add a comment below the main post of your issue saying: `Hi @brainhackorg/project-monitors my project is ready!`
- [ ] Twitter-sized summary of your project pitch.

**Comment by Remi-Gau** [comment]
@RickReddy 
your project is displayed on the website

**Comment by Remi-Gau** [comment]
http://brainhack.org/global2023/projects/
